saleor/graphql/invoice/types.py

```diff
@@ -8,11 +8,15 @@
 
 
 class Invoice(ModelObjectType[models.Invoice]):
-    number = graphene.String()
-    external_url = graphene.String()
-    created_at = graphene.DateTime(required=True)
-    updated_at = graphene.DateTime(required=True)
-    message = graphene.String()
+    number = graphene.String(description="Invoice number.")
+    external_url = graphene.String(description="url to view an invoice.")
```

Comment:
Nitpick: would be good to keep it consistent and start each description with a capital letter.